internal/graphapi/subscriber_test.go

```
@@ -149,50 +150,56 @@ func (suite *GraphTestSuite) TestMutationCreateSubscriber() {
 	t := suite.T()
 
 	testCases := []struct {
-		name            string
-		email           string
-		ownerID         string
-		setUnsubscribed bool
-		client          *openlaneclient.OpenlaneClient
-		ctx             context.Context
-		wantErr         bool
+		name             string
+		email            string
+		ownerID          string
+		setUnsubscribed  bool
+		client           *openlaneclient.OpenlaneClient
+		ctx              context.Context
+		wantErr          bool
+		expectedAttempts int64
 	}{
 		{
-			name:            "happy path, new subscriber",
-			email:           "c.stark@example.com",
-			setUnsubscribed: true, //unsubscribe the subscriber to test for re-creation
-			client:          suite.client.api,
-			ctx:             testUser1.UserCtx,
-			wantErr:         false,
+			name:             "happy path, new subscriber",
+			email:            "c.stark@example.com",
+			setUnsubscribed:  true, //unsubscribe the subscriber to test for re-creation
+			client:           suite.client.api,
+			ctx:              testUser1.UserCtx,
+			wantErr:          false,
+			expectedAttempts: 1,
 		},
 		{
-			name:    "happy path, duplicate subscriber but original was unsubscribed",
-			email:   "c.stark@example.com",
-			client:  suite.client.api,
-			ctx:     testUser1.UserCtx,
-			wantErr: false,
+			name:             "happy path, duplicate subscriber but original was unsubscribed",
+			email:            "c.stark@example.com",
+			client:           suite.client.api,
+			ctx:              testUser1.UserCtx,
+			wantErr:          false,
+			expectedAttempts: 2,
 		},
 		{
-			name:    "happy path, duplicate subscriber, case insensitive",
-			email:   "c.STARK@example.com",
-			client:  suite.client.api,
-			ctx:     testUser1.UserCtx,
-			wantErr: true,
```

Can we add another test in here and ensure these pass:

1. If a subscriber is active (already verified) - we should return the error that the subscriber already exists
2. If a subscribe is not active/email is not verified we should resent.